include/cantera/thermo/MixtureFugacityTP.h

```diff
@@ -571,6 +562,19 @@ class MixtureFugacityTP : public ThermoPhase
 
     //! Temporary storage for dimensionless reference state entropies
     mutable vector_fp m_s0_R;
+
+public:
+    //! Omega constants: a0 (= omega_a) and b0 (= omega_b) values used in Peng-Robinson equation of state
+    /*!
+     *  These values are calculated by solving P-R cubic equation at the critical point.
+     */
+    static const double omega_a;
+
+    //! Omega constant for b
+    static const double omega_b;
+
+    //! Omega constant for the critical molar volume
+    static const double omega_vc;
```

Comment:
@speth They are, but they are used in the cubic solver.  Simply deleting them leads to build errors.  I suspect I would have to pass them in as inputs, if we want to have them undefined at the `MixtureFugacityTP` level.  

Is that preferred?

Reply:
If these variables need to be used by a method defined in `MixtureFugacityTP`, then they need to be regular members, declared in the base class only, not `static const` variables declared both here and in the child classes. With the current implementation, `MixtureFugacityTP::omega_b` and `PengRobinson::omega_b` are different variables, where the methods of `MixtureFugacityTP` can only see the variable `MixtureFugacityTP::omega_b` (which is set to 0.0).

Rather than doing this, though, I see that `solveCubic` barely uses the values of `omega_b` and `omega_vc` at all -- they are only used at the very end, in a way that makes the return value `nSolnValues` more cryptic than necessary. That is, if we eliminate the logic for returning negative values of `nSolnValues`, you can work out which branch the returned solution is on later.

Reply:
@speth Since omega_a, omega_b variables are specific to the child classes, I have removed their definitions from the MixtureFugacityTP class.  These values are only needed to calculate critical pressure pc at the very end. These critical proportiesare now calculated in the respective child class functions and passed as parameters to the solveCubic() function in MixtureFugacityTP  class.